src/components/widgets/text_editors/MultiFormatTextEditor.js

```diff
@@ -209,7 +209,14 @@ class MultiFormatTextEditor extends Component {
       return null;
     }
 
-    const {onChange, placeholder, value, localize: _} = this.props;
+    const {onChange, localize: _} = this.props;
+    let {placeholder, value} = this.props;
+
+    if (value.startsWith('Click to enter') && value.endsWith('title')) {
```

Comment:
or @alexcjohnson would you know of a better way for me to get that default string when title's not set?

Reply:
Mmm, that way isn't going to work with i18n anyway. Is it possible to compare the value from `fullLayout` with the value from `layout`? If they're different, I think that's a robust signal that you have the placeholder. And if you explicitly typed in the placeholder, then it's not really a placeholder anymore :)

Reply:
If we're not comfortable using the plotly.js placeholder verbatim, I think we'd have to explicitly make and localize a new placeholder for each attribute that uses this. Doesn't seem worth it to me, frankly; I'd either use the one from plotly.js or use a standard placeholder for every text entry box.

Reply:
yeah, we could use those plotly.js strings verbatim, it's easier for localization too, like you mention, so I'll put those strings as placeholders then, not values